lua/pac3/core/client/parts/event.lua

```diff
@@ -723,6 +723,122 @@ PART.OldEvents = {
 		end,
 	},
 
+
+	is_no_draw = {
+		callback = function(self, ent)
+			ent = try_viewmodel(ent)
+			return ent:GetNoDraw()
+		end,
+	},
+	
+	is_no_target = {
+		callback = function(self, ent)
+			ent = try_viewmodel(ent)
+			return ent:IsFlagSet( FL_NOTARGET )
+		end,
+	},
+
+
+	// This is a function to grab any and all network variables and try to run them with the correct type, as it's possible to have GetNWInt("Pac", 1) and "GetNWString("Pac" ValueHere") 
+	// This code is designed to try and run things correctly and detect specific types of Network Variables, Since GetNWVar does not have the varable display in the table, but GetNW2var does.
+	// It's also useful if a player doesn't know what the displayed function is as a Networked Variable.
```

Comment:
The // needs to be replaced with -- here and at line 766

Reply:
perplexed to ask why? just for view? both -- and // do the same thing in gmod lua?

Reply:
It breaks code editors since // is nonstandard lua.

Reply:
gotcha